data.filter is not a function

```
14:35:18 [ERROR] [Magellan] TypeError: data.filter is not a function
14:35:18 at Object._normalize (//node_modules/guacamole/src/browsers.js:221:8)
14:35:18 at SauceBrowsers._fetch.then (/node_modules/guacamole/src/browsers.js:487:34)
```

I think the issue might be that when `SauceLabs` does not return an array in the following call

```javascript
SauceBrowsers._fetch()
      .then((data) => {
        SauceBrowsers._haveCachedSauceBrowsers = true;
        browsers = SauceBrowsers._normalize(data);
```

it will then try to call `.filter` on an object or null object at

```javascript
_normalize: (data) => {
    const overallResult = data
      .filter((browser) =>
```

--- a/src/browsers.js
+++ b/src/browsers.js
@@ -217,6 +217,9 @@ const SauceBrowsers = {
   },
 
   _normalize: (data) => {
+    if (!Array.isArray(data)) {
+      throw new Error(`Invalid Response: ${data}`);
+    }
     const overallResult = data
       .filter((browser) =>
         browser.automation_backend === "webdriver" || browser.automation_backend === "appium"

--- a/src/browsers.js
+++ b/src/browsers.js
@@ -218,7 +218,7 @@ const SauceBrowsers = {
 
   _normalize: (data) => {
     if (!Array.isArray(data)) {
-      throw new Error(`Invalid Response: ${data}`);
+      throw new Error(`Invalid Response: ${JSON.stringify(data)}`);
     }
     const overallResult = data
       .filter((browser) =>

--- a/test/normalize.js
+++ b/test/normalize.js
@@ -78,4 +78,10 @@ describe("normalizer", () => {
     expect(() => Browsers._normalize(response)).to.throw(`Invalid Response: ${response}`);
   });
 
+  it("handles and logs a non-array, object response", () => {
+    const response = { error: "something went wrong" };
+    const expected = `Invalid Response: ${JSON.stringify(response)}`;
+    expect(() => Browsers._normalize(response)).to.throw(expected);
+  });
+
 });

--- a/test/normalize.js
+++ b/test/normalize.js
@@ -73,4 +73,9 @@ describe("normalizer", () => {
     expect(ipad).to.have.property("deviceOrientation", "landscape");
   });
 
+  it("handles and logs a non-array response", () => {
+    const response = null;
+    expect(() => Browsers._normalize(response)).to.throw(`Invalid Response: ${response}`);
+  });
+
 });

